Extensions/dnSpy.Debugger/dnSpy.Debugger/Attach/AppCommandLineArgsHandler.cs

```diff
@@ -20,32 +20,72 @@
 using System;
 using System.ComponentModel.Composition;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Threading;
+using System.Threading.Tasks;
 using dnSpy.Contracts.App;
+using dnSpy.Contracts.Debugger;
 using dnSpy.Contracts.Debugger.Attach;
 
 namespace dnSpy.Debugger.Attach {
 	[Export(typeof(IAppCommandLineArgsHandler))]
 	sealed class AppCommandLineArgsHandler : IAppCommandLineArgsHandler {
 		readonly Lazy<AttachableProcessesService> attachableProcessesService;
+		readonly Lazy<DbgManager> dbgManager;
 
 		[ImportingConstructor]
-		AppCommandLineArgsHandler(Lazy<AttachableProcessesService> attachableProcessesService) =>
+		AppCommandLineArgsHandler(Lazy<AttachableProcessesService> attachableProcessesService, Lazy<DbgManager> dbgManager) {
 			this.attachableProcessesService = attachableProcessesService;
+			this.dbgManager = dbgManager;
+		}
 
 		public double Order => 0;
 
+		[DllImport("kernel32.dll")]
+		private static extern bool SetEvent(IntPtr hEvent);
+		[DllImport("kernel32.dll")]
+		private static extern bool CloseHandle(IntPtr hObject);
+		private async Task BreakOnAttach(AttachableProcess process) {
+			TaskCompletionSource<bool> isDebuggingChangedSrc = new();
+			TaskCompletionSource<bool> isRunningChangedSrc = new();
+
+			EventHandler? IsDebuggingHandler=null;
+			EventHandler? IsRunningHandler = null;
+			var mgr = dbgManager.Value;
+
+
+			IsDebuggingHandler = ( _, _) => { if (mgr.IsDebugging == true) isDebuggingChangedSrc.SetResult(true); };
+			IsRunningHandler = ( _, _) => { if (mgr.IsRunning == true) isRunningChangedSrc.SetResult(true); };
```

Comment:
These definitions could be inlined to the variable declarations.
Furthermore, avoid forcing one-liners like this. It's okay to define it as a multi-line anonymous method.